### PR TITLE
[sc-68952] Add items to zone by batch

### PR DESCRIPTION
**Pull request by qcastel**
This PR goes to pair with https://github.com/dataiku/dip/pull/14461

### CODE REVIEW

**Comment by qcastel** [comment]
[sc-68952]

File: dataikuapi/dss/flow.py [hunk 1]
```diff
@@ -338,6 +338,22 @@ def add_item(self, obj):
...
 
+    def add_items(self, items):
+        """
+        Adds items to this zone.
+
+        The items will automatically be moved from its existing zone. Additional items may be moved to this zone
+        as a result of the operation (notably the recipe generating `obj`).
+
+        :param object items: A :class:`dataikuapi.dss.dataset.DSSDataset`, :class:`dataikuapi.dss.managedfolder.DSSManagedFolder`,
+                           or :class:`dataikuapi.dss.savedmodel.DSSSavedModel` to add to the zone
```
Comment (marchelleboid) [inline dataikuapi/dss/flow.py:349]: A few things here should be updated now that it's a list of items, probably operation -> operations, that phrase about the recipe, and the param description should be updated to say "a list of ..."

**Review by marchelleboid — CHANGES_REQUESTED** [review]

**Review by marchelleboid — APPROVED** [review]
LGTM with a couple final doc changes

**Review by nikopol — APPROVED** [review]
LGTM. You should add an exemple in [dss doc](https://doc.dataiku.com/dss/latest/python-api/flow.html#creating-a-zone-and-adding-items-in-it) too I guess.

**Review by mhoarau — CHANGES_REQUESTED** [review]

File: dataikuapi/dss/flow.py [hunk 2]
```diff
@@ -338,6 +338,22 @@ def add_item(self, obj):
...
                                   body=self.flow._to_smart_ref(obj))
 
+    def add_items(self, items):
+        """
+        Adds items to this zone.
+
+        The items will automatically be moved from their existing zones. Additional items may be moved to this zone
+        as a result of the operations (notably the recipe generating the `items`).
+
+        :param list items: A list of objects, either :class:`dataikuapi.dss.dataset.DSSDataset`, :class:`dataikuapi.dss.managedfolder.DSSManagedFolder`,
```
[the remark this answers is not in the document]
Comment (qcastel) [reply dataikuapi/dss/flow.py:348]: If you look at the same feature in the UI, you can't move recipe directly. However a recipe will be moved to the zone if the output dataset is moved to the zone. At least thats what I experienced in the UI so far

**Review by mhoarau — APPROVED** [review]

**Pull request merged**